sqlmesh/core/plan/definition.py

```diff
@@ -494,7 +490,6 @@ def has_unmodified_unpromoted(self) -> bool:
     def _add_restatements(self) -> None:
         def is_restateable_snapshot(snapshot: Snapshot) -> bool:
             if not self.is_dev and snapshot.disable_restatement:
-                logger.debug("Restatement is disabled for model '%s'.", snapshot.name)
```

Comment:
Why are these debug messages removed?

Reply:
didn't seem very useful

Reply:
@izeigerman I'm guessing you added these so making sure you saw this

Reply:
I think it was useful when we were ignoring non-restateable models rather than failing. Since we now fail if the model is non-restateable, this log message is indeed not very useful.